blosc/blosc.c

```diff
@@ -1455,7 +1467,7 @@ int blosc_decompress(const void *src, void *dest, size_t destsize)
 int blosc_getitem(const void *src, int start, int nitems, void *dest)
 {
   uint8_t *_src=NULL;               /* current pos for source buffer */
-  uint8_t version, versionlz;       /* versions for compressed header */
+  uint8_t version, compressor_version; /* versions for compressed header */
```

Comment:
For being consistent with blosc convention for names(e.g. `compname`, `compformat`), I'd prefer using `compversion` for this variable.